e2e/tests/stress_cpu.robot

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Documentation    Negative Test Cases
+Resource    ../keywords/common.resource
+Resource    ../keywords/node.resource
+Resource    ../keywords/volume.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+
+Stress Volume Node CPU When Replica Is Rebuilding
+    Given Create a volume with 5 GB and 3 replicas
+    And Write data to the volume
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on volume node to trigger replica rebuilding
+        And Stress the CPU of all volume nodes
```

Comment:
Since there is only **one** volume created, why we say Stress the CPU of **all** volume node**s** here?

Reply:
This enhancement involves utilizing 'volume_list' instead of 'single volume_name'. I don't believe we require an additional keyword to manage the plural; it serves the same purpose. Thus, in the scenario of a single volume, it strains the CPU of that specific volume node.